templates/events/manager/events/create_update.html

```diff
@@ -236,12 +233,28 @@ <h1 class="h2" tabindex="0">Create an Event</h1>
             <small class="form-text text-muted mt-0" id="help-label-tags" tabindex="-1">Provide keywords that describe this event for better searchability.</small>
           </div>
           <div class="col-md-5 col-xl-4 mb-2 mb-md-0">
-            {% render_field form.tags class="form-control" aria_describedby="help-label-tags" %}
+            <div class="row no-gutters">
+              <div class="col-10">
+                <input type="text" id="event-tags-typeahead" class="form-control" autocomplete="false">
+              </div>
+              <div class="col-2">
+                <a id="add-new-tag" href="#" class="btn btn-primary" style="display: none;">&plus;</a>
+              </div>
+            </div>
+            {% render_field form.tags class="form-control d-none" aria_describedby="help-label-tags" %}
             {{ form.tags.errors }}
             <small class="form-text text-muted" tabindex="0">Type tags separated by commas in this field, e.g. <strong>football, UCF Sports, Knights</strong>.<br><br>Please note that tags are case-insensitive.</small>
           </div>
           <div class="col-md-4 col-xl-4">
             <ul id="event-tags-selected" class="list-unstyled">
+              {% for tag in event.tags.all %}
+              <li class="list-inline-item" data-tag-text="{{ tag.name }}">
+                <span class="badge badge-pill badge-default">
+                  <a href="#" class="text-inverse" alt="Remove this tag" title="Remove this tag">&times</a>
```

Comment:
forgot about this template being used for the update view 🤦 
we can just update this remove tag link the same way as above
```suggestion
                  <a href="#" class="selected-remove" alt="Remove this tag" title="Remove this tag"><span class="fa fa-times mr-1" aria-hidden="true"></span></a>
```

Reply:
Updated!